lib/perl/Genome/WorkflowBuilder/DAG.pm

```diff
@@ -61,6 +61,93 @@ sub add_link {
 # Public Methods
 # ------------------------------------------------------------------------------
 
+sub execute {
+    my $self = shift;
+
+    my %p = Params::Validate::validate(@_, {
+        inputs => {type => HASHREF},
+    });
+
+    my $inputs = {%{$self->constant_values}, %{$p{inputs}}};
+
+    my $backend = $ENV{GENOME_WORKFLOW_BUILDER_BACKEND} || 'workflow';
```

Comment:
This is a duplication of the default value since it's also set in the `Genome::Env` module.